### PR TITLE
Translations: Highlights Onboarding Experiment

### PR DESCRIPTION
**Pull request by nalcalag**
Task/Issue URL: https://app.asana.com/0/72649045549333/1208294136875672/f

### Description
Translations for new strings added in highlights experiment

### Steps to test this PR (Optional)

- [ ] Smoke test

### CODE REVIEW

**Comment by nalcalag** [comment]
* **#5004** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5004?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nalcalag and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/src/main/res/values-bg/strings.xml [hunk 1]
```diff
@@ -17,8 +17,8 @@
 <!-- smartling.entity_escaping = false -->
 <!-- smartling.instruction_attributes = instruction -->
 <resources xmlns:tools="http://schemas.android.com/tools">
-    <string name="appName" translatable="false">DuckDuckGo</string>
-    <string name="appDescription" translatable="false">DuckDuckGo</string>
```
Comment (joshliebe) [inline app/src/main/res/values-bg/strings.xml:21]: Not sure why we removed this?

**Review by joshliebe — APPROVED** [review]
LGTM, just the one comment: https://github.com/duckduckgo/Android/pull/5004/files#r1764715727

File: app/src/main/res/values-bg/strings.xml (hunk 1 above)
Comment (nalcalag) [reply app/src/main/res/values-bg/strings.xml:21]: Good catch! This change came from Smartling, so it's not related to my changes. To fix it, I'll merge my branch first and then create a new Smartling job in a different branch so it gets updated remotely in Smartling as well.

**Pull request merged**